qiskit/circuit/quantumcircuit.py

```diff
@@ -45,9 +47,17 @@
 
 try:
     import pygments
-    from pygments.formatters import Terminal256Formatter  # pylint: disable=no-name-in-module
+    from pygments.formatters import (
+        Terminal256Formatter,
+    )  # pylint: disable=no-name-in-module
     from qiskit.qasm.pygments import OpenQASMLexer  # pylint: disable=ungrouped-imports
+<<<<<<< HEAD
+    from qiskit.qasm.pygments import (
+        QasmTerminalStyle,
+    )  # pylint: disable=ungrouped-imports
+=======
```

Comment:
@1ucian0 There's a lot of merge conflicts in this file. Should I resolve them?

Reply:
Ups. I missed then. Please go ahead